ui/src/app/tasks-jobs/tasks/launch/builder/builder.component.html

```diff
@@ -31,7 +33,7 @@
             <div class="line-toggle">
               <a class="toggle" (click)="state.platform = !state.platform">
                 <clr-icon shape="angle" [class.active]="state.platform"></clr-icon>
-                Platform
+                {{ 'tasks.launch.builder.platform' | translate }}
```

Comment:
`Platform` text visible on other pages.
Can we move this to `commons` translations ?